examples/x402-facilitator-plugin-example/.env.example

```
@@ -0,0 +1,3 @@
+REDIS_URL=redis://redis:6379
+API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_KEY key should go before the REDIS_URL key

(UnorderedKey)